src/components/account.tsx

```diff
@@ -0,0 +1,196 @@
+import { BigNumber, ERC20TokenWrapper } from '0x.js';
+import { Web3Wrapper } from '@0xproject/web3-wrapper';
+import { Button, Column, Columns, Content, Icon, Subtitle, Table } from 'bloomer';
+import * as _ from 'lodash';
+import * as React from 'react';
+
+import { ETHER_TOKEN, tokensByNetwork } from '../tokens';
+
+interface Props {
+    web3Wrapper: Web3Wrapper;
+    erc20TokenWrapper: ERC20TokenWrapper;
+    toastManager: { add: (msg: string, appearance: {}) => void };
+}
+
+interface Token {
+    name: string;
+    address: string;
+    symbol: string;
+    decimals: number;
+}
+interface TokenBalanceAllowance {
+    token: Token;
+    balance: BigNumber;
+    allowance: BigNumber;
+    tradeable: boolean;
+}
+
+interface AccountState {
+    balances: { [address: string]: TokenBalanceAllowance[] };
+    selectedAccount: string;
+}
+
+export class Account extends React.Component<Props, AccountState> {
+    constructor(props: Props) {
+        super(props);
+        this.state = { balances: {}, selectedAccount: '' };
+        void this.fetchAccountDetailsAsync();
+        setInterval(() => {
+            void this.checkAccountChangeAsync();
+        }, 2000);
+    }
+    public fetchAccountDetailsAsync = async () => {
+        const { web3Wrapper, erc20TokenWrapper } = this.props;
+        const { balances } = this.state;
+        const addresses = await web3Wrapper.getAvailableAddressesAsync();
+        const address = addresses[0];
+        if (!address) {
+            return;
+        }
+        const networkId = await web3Wrapper.getNetworkIdAsync();
+        const tokens = tokensByNetwork[networkId];
+        // Fetch all the Balances for all of the tokens
+        const allBalancesAsync = _.map(
+            tokens,
+            async (token: Token): Promise<TokenBalanceAllowance> => {
+                const balance = await erc20TokenWrapper.getBalanceAsync(token.address, address);
+                const allowance = await erc20TokenWrapper.getProxyAllowanceAsync(token.address, address);
+                const numberBalance = new BigNumber(balance);
+                return { token, balance: numberBalance, allowance, tradeable: true };
```

Comment:
All boolean variables must start with `is`, `should`, `does`, `has`, etc... `isTradeable`. Also what does tradeable mean? Does it mean it's not ETH?